pkg/ddevapp/config_test.go

```diff
@@ -214,6 +214,12 @@ func TestConfigCommandInteractiveCreateDocrootDenied(t *testing.T) {
 	// Set up tests and give ourselves a working directory.
 	assert := asrt.New(t)
 
+	noninteractive := "DRUD_NONINTERACTIVE"
```

Comment:
I was confused by this env name variable, assuming it was the value, not the name. Silly nit, but maybe name it noninteractiveEnv?